src/main/java/uk/gov/companieshouse/service/PaymentService.java

```diff
@@ -0,0 +1,49 @@
+package uk.gov.companieshouse.service;
+
+import org.springframework.stereotype.Service;
+import uk.gov.companieshouse.model.dto.PaymentDescriptionValues;
+import uk.gov.companieshouse.model.dto.PaymentGetResponse;
+import uk.gov.companieshouse.model.dto.PaymentItem;
+import uk.gov.companieshouse.model.dto.PaymentLinks;
+
+import java.util.List;
+
+import static uk.gov.companieshouse.model.Constants.*;
```

Comment:
Even though it's annoying to do, I think it's better to specify each import we actually want/use

```suggestion
import static uk.gov.companieshouse.model.Constants.PAYMENT_DESCRIPTION_IDENTIFIER;
import static uk.gov.companieshouse.model.Constants.PAYMENT_KIND;
import static uk.gov.companieshouse.model.Constants.PAYMENT_PRODUCT_TYPE;
import static uk.gov.companieshouse.model.Constants.PAYMENT_AMOUNT;
import static uk.gov.companieshouse.model.Constants.PAYMENT_AVAILABLE_PAYMENT_METHOD;
import static uk.gov.companieshouse.model.Constants.PAYMENT_CLASS_OF_PAYMENT;
import static uk.gov.companieshouse.model.Constants.PAYMENT_ITEM_KIND;
import static uk.gov.companieshouse.model.Constants.PAYMENT_RESOURCE_KIND;
```